### PR TITLE
feat: invite by email who is out of system.

### PR DESCRIPTION
**Pull request by naftalima**
## Objective and Motivation
- A Trainer can invite an opponent by email if they are not yet in the system.

### CODE REVIEW

**Comment by naftalima** [comment]
AttributeError: 'NoneType' object has no attribute 'get_host'
image

**Comment by naftalima** [comment]
This email template is the [Django default.](https://github.com/django/django/blob/main/django/contrib/admin/templates/registration/password_reset_email.html)

File: backend/templates/registration/password_reset_subject.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Open-Chat password reset
```
Comment (rsarai) [inline backend/templates/registration/password_reset_subject.txt:1]: What's Open-Chat?

Comment (naftalima) [reply backend/templates/registration/password_reset_subject.txt]: IDK- It was on the template. I'll delete this and rename everything.

**Review by naftalima — COMMENTED** [review]
what?

**Review by rsarai — APPROVED** [review]

**Pull request merged**